.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,11 @@
+**Is your feature request related to a problem?**
```

Comment:
This format doesn't seem very useful, could we re-write to provide the needed details for the documentation team, is the following all we need?

```
### Documentation update for OpenSearch Security Plugin
An update to the documentation is needed, see the related pull request for the changes that have been introduced.

Pull request:
```

Reply:
Note; it might be worthwhile to see about using SED or another utility to allow env variables to be updated in the template file so the workflow can provide more information

Reply:
Yeah I had tried using that but faced some issues as far as I remember. I will try again to see how it can be made better with more information.